### PR TITLE
Some talking toys now audibly chatter

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax, Nienhaus
rscdel: Unfortunately, the Belowyarn toys have been withdrawn from distribution
due to Assistants choking on the batteries. Centcom has sent their
condolences to the battery manufacturing plant.
rscadd: We are introducing Oh-cee the original content skeleton figurines.
Pull its string, and hear it say a variety of phrases. Not suitable for
infants or assistants under 36 months.
tweak: Note that pulling the string of all talking toys is now visible
to others.
rscadd: Talking toys produce a lot more chatter than they did before. So do some skeleton "tongues".
/:cl:

Soviet Commander Braxton Cherry is pleased with his arcade prize

With thanks to Nienhaus for doing the sprite.
- The toy AI has also been made a "talking" toy because it works the
  same as the others.
- chatter() is a global proc, not limited to just talking action figures

EDIT:
### The Journey of this PR
- Notice that skeletons use the same font spans as Undertale characters.
- Create small project to make chatter when skeletons species talk, rationale, it'd be a fun additional nod, a recent PR to make voices when people talk was merged for a time, and the skeleton species only appears in wizard during murderboning lich anyway.
- People dislike the voices, feel too noisy
- Someone suggests adding the voice to arcade prizes, this idea is less controversial. Chatter is made quieter.
- The toys are merged, with fairly poor coder sprites. They prove mildly popular in-game, some problems with people spamming the toy and not knowing who's doing it
- Five hours later, are speed reverted while author is sleeping with the justification that it breaks the build (for a version we barely support)
- New sprites are made, better ss13-style phrases used, visual messages made by pulling the string, meme suicides added
- After multiple compromises, working with multiple people to find a pleasant compromise for still using the chatter code, is deemed: shitty autism, forced, shoe horned, and covered with unhelpful annotations from the usual suspects
- 10/10 would coderbus again
- Skeleton toy now consists of 80% skeleton puns, all use of talking toys produce a visible message to allow lynching
- When I try to change the title to more accurately describe the content, accused of "hiding the bullshit"
- Now tagged "universally disliked"

### CODE REVIEW

**Comment by Man-Shroom-** [comment]
no undertale ref

bad game

**Comment by Durkel** [comment]
How many times do we have to try and shoe horn in these fucking undermeme references?

**Comment by PKPenguin321** [comment]
yeah this is getting really forced

**Comment by Incoming5643** [comment]
No

**Comment by bgobandit** [comment]
so many commits

**Comment by Shadowlight213** [comment]
No. The voice thing has been denied multiple times.

**Comment by Armhulen** [comment]
it isn't even funny, no thanks

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -732,6 +729,57 @@
...
+	name = "skeleton action figure"
+	desc = "An action figure modeled after 'Oh-cee', the original content \
+		skeleton.\nNot suitable for infants or assistants under 36 months \
+		of age."
+	icon_state = "skeletonprize"
+	attack_verb = list("boned", "dunked on", "worked down to the bone")
+
+	var/phomeme
+
+	var/list/papyrus_messages = list(
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: no

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -732,6 +729,57 @@
...
+		"That's the disposal bin. Feel free to visit it at any time.",
+		"I can't just let anyone ERP with me, I'm a skeleton with standards!",
+		"Sorry. Can't talk. I'm busy being popular on a video game.",
+		"You can't spell 'robust' without several letters from my name!!!")
+	var/list/sans_messages = list(
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: no

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -732,6 +729,57 @@
...
+/obj/item/toy/talking/skeleton/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is trying to commit \
+		suicide with \the [src].</span>")
+
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		H.set_species(/datum/species/skeleton)
+
+	set_phomeme("sans")
+	toy_talk(user, "You feel like you're going to have a bad time.")
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: no

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -732,6 +729,57 @@
...
+
+/obj/item/toy/talking/skeleton/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is trying to commit \
+		suicide with \the [src].</span>")
+
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		H.set_species(/datum/species/skeleton)
+
+	set_phomeme("sans")
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: i added this for comic sans memes, not shitty undertale autism

**Comment by AdamElTablawy** [comment]
is undertale a good game? yes

do you need to fucking shoehorn in every fucking reference? NO

**Comment by coiax** [comment]
> Wow guys, it's two toys and you're acting like they've replaced the main players in the game with undertale.

**Comment by Incoming5643** [comment]
I was almost going to give you the benefit of the doubt on this and then I saw that goddamn changelog :(

**Comment by coiax** [comment]
?

**Comment by ShadowDeath6** [comment]
At least it's visible when you pull the string so you can lynch people who spam it. Maybe less undermemes but more regular skeleton puns

**Comment by AdamElTablawy** [comment]
yeah skeleton puns would be fine just remove the goddamn references

**Comment by Shadowlight213** [comment]
Remove the sound files too. It's needless resource bloat.

**Comment by duncathan** [comment]
the sound engine is the whole goddamn point and there's nothing wrong with it or the sounds

**Comment by Shadowlight213** [comment]
Ok

**Comment by coiax** [comment]
Give me some more skeleton toy phrases, I'll put them in the pool and pick a random span for them. I'll also replace the existing phomeme pool with lower quality, that should offer no reduction in audio quality but a greatly reduced size.

If anyone could point out some phomeme samples for the AI toy, griffin and eagle that would be great too.

**Comment by AdamElTablawy** [comment]
just make it shout bad puns

**Comment by AdamElTablawy** [comment]
why was the skeleton such a bad liar because you can see right through him

when does a skeleton laugh when something tickles his funny bone

why couldn't the skeleton win the beauty contest because he had no body

what do you call a skeleton in the winter a numbskull

what did the skeleton say before eating bone appetit

what type of art do skeletons like skulltures

what instrument do skeletons play the tromBONE

wHY ARE SKELETONS ALWAYS SO CALM BECAUSE NOTHING GETS UNDER THEIR SKIN

hOW DID THE SKELETON KNOW IT WAS GOING TO RAIN HE COULD FEEL IT IN HIS BONES

WHY DID THE SKELETON GO TO THE HOSPITAL TO GET HIS GHOUL STONES REMOVED

WHY CANT SKELETONS PLAY MUSIC IN CHURCHES BECAUSE THEY HAVE NO ORGANS

**Comment by coiax** [comment]
These are terrible. :+1:

**Comment by ShadowDeath6** [comment]
rattle me bones

**Comment by Iamgoofball** [comment]
Remove the span classes and remove all undertale references and I'm okay with this

**Comment by ShadowDeath6** [comment]
Papyrus font looks nice though

**Comment by Iamgoofball** [comment]
No. Remove it.

**Comment by ShadowDeath6** [comment]
Remove font discrimination >:^(

**Comment by Iamgoofball** [comment]
No, you just want to keep your shitty undertale memes. Fuck off.

**Comment by MrStonedOne** [comment]
Seriously guys, knock it the fuck off.

They had a merged pr they spent alot of time on speed reverted over a bullshit reason and now everybody is trying to pretend it's still merged while rejecting any new pr to re-add it or similar shit.

Give the poor guy a break for fucks sake.

**Comment by Iamgoofball** [comment]
I am perfectly fine with the sound system and the other PR shit.

I just don't want the undertale shit. That's all I care about.

**Comment by TechnoAlchemisto** [comment]
Although I did like UT, i'd prefer references to it stayed more or less out of this game

**Comment by Niknakflak** [comment]
>techno says he prefer references out of the game

>he added his own references to dark souls


Faggot hypocrite detected

**Comment by Niknakflak** [comment]
Faggot hypocrite detected

**Comment by Incoming5643** [comment]
W E  
A L R E A D Y  
H A V E  
A N  
U N D E R T A L E  
R E F E R E N C E
T H A T S
L E S S
O B N O X I O U S

**Comment by AdamElTablawy** [comment]
merge this specifically to spite niknakflak

**Comment by coiax** [comment]
@WJohn this doesn't need the Revert/Removal tag, the CL is just providing an ingame reason to why the original toys disappeared so suddenly, they're already gone.

**Comment by coiax** [comment]
@Dorsisdwarf I wouldn't say... universally disliked. Based at least on some of the comments that have been stated subsequently.

**Comment by Iamgoofball** [comment]
>renaming the PR to hide your bullshit

seriously JUST FUCKING REMOVE THE UNDERTALE LINES, FUCK

**Comment by Iamgoofball** [comment]
https://github.com/coiax/-tg-station/pull/4
THere, I did it for you.

If you refuse to merge this than the competition PR will likely be merged instead
https://github.com/tgstation/-tg-station/pull/17630

**Comment by KorPhaeron** [comment]
You should probably do what goof did so it will get merged, I doubt anyway wants to touch it as is

**Comment by ChangelingRain** [comment]
I'm going to prod coiax into adding a ton more skele-puns so that the 7 ref lines are mostly hidden by puns and THEN merge it.

**Comment by ChangelingRain** [comment]
Now at 7/38 jokes being references, which is okay and if that's still too high a chance for a reference I'll just prod coiax into adding more jokes

**Comment by ChangelingRain** [comment]
If necessary, I'll just google skeleton jokes and add as many as possible.

**Comment by Incoming5643** [comment]
Why

**Pull request merged**